Packs/FeedAzure/Integrations/FeedAzure/FeedAzure.py

```diff
@@ -277,7 +278,10 @@ def fetch_indicators_command(client: Client, limit: int = -1) -> Tuple[List[Dict
         indicators.append({
             'value': indicator['value'],
             'type': indicator['type'],
-            'fields': {'region': indicator.get('azure_region')},
+            'fields': {
+                'region': indicator.get('azure_region'),
+                'tags': tags
```

Comment:
is it ok we pass here empty list in case no tags were provided? it looks ok in the ui on the indicator layout?

Reply:
Looks great.
![image](https://user-images.githubusercontent.com/45915502/77900156-4cece880-7286-11ea-931f-774198cc2ff8.png)